apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -5,11 +5,13 @@ import SectionProgressToggle from '@cdo/apps/templates/sectionProgress/SectionPr
 import VirtualizedDetailView from './VirtualizedDetailView';
 import VirtualizedSummaryView from './VirtualizedSummaryView';
 import SummaryViewLegend from './SummaryViewLegend';
+import ProgressLegend from '@cdo/apps/templates/progress/ProgressLegend';
```

Comment:
outside the scope of this PR, but now that we have 2 versions - a summary view legend and a detail view legend we should maybe re-name this one to be more specific?

Reply:
I'm mixed on this, but you're right there's a bunch of naming we could probably make more clear. 
"Progress" throughout our code refers to the bubbles so I'm inclined to leave the bubble table as `ProgressLegend` (https://github.com/code-dot-org/code-dot-org/tree/staging/apps/src/templates/progress)
There is actually a `SummaryProgressTable` and a `VirtualizedSummaryView` and a `SummaryViewLegend` that only refers to the `VirtualizedSummaryView`.
We have /sectionProgress/ and /Progress/ in /templates/, but in storybook we only have /Progress category. 🤷‍♀️ 

I'll add an item to our list to look at naming after this sprint.